sdk/monitor/azquery/README.md

```diff
@@ -145,6 +147,7 @@ comments.
 [default_cred_ref]: https://github.com/Azure/azure-sdk-for-go/tree/main/sdk/azidentity#defaultazurecredential
 [example_logs_client]: https://pkg.go.dev/github.com/Azure/azure-sdk-for-go/sdk/monitor/azquery#example-NewLogsClient
 [example_metrics_client]: https://pkg.go.dev/github.com/Azure/azure-sdk-for-go/sdk/monitor/azquery#example-NewMetricsClient
+[example_metrics_batch_client]: https://pkg.go.dev/github.com/Azure/azure-sdk-for-go/sdk/monitor/azquery@v1.2.0-beta.1#example-NewMetricsBatchClient
```

Comment:
Ultimately you want to link to the example without specifying a version so clickers always land on the latest stable docs. But the example doesn't exist in the stable docs yet, so you have some options here:
1. remember to remove the version from this link after the 1.0 release
2. remove the version now. The link will take clickers to the top of the docs today and, after the release, to the example
3. use an aka link with fragment e.g. `https://aka.ms/azsdk/go/monitor/azquery#example-NewMetricsBatchClient`

I like option 2

Reply:
I think you're right. I like option 2 the best. Changed in [b9b76fa](https://github.com/Azure/azure-sdk-for-go/pull/22243/commits/b9b76fa15f241675fd7abf6fb0a063a5faa8b1f4)